core/src/main/java/org/infinispan/manager/DefaultCacheManager.java

```diff
@@ -809,7 +809,7 @@ private void internalStop() {
          }
 
          // We can stop the manager
-         log.infof("Stopping cache manager %s on %s", configurationManager.getGlobalConfiguration().transport().nodeName(), getAddress());
+         log.debugf("Stopping cache manager %s", identifierString());
```

Comment:
not sure if it is useful to store `identifierString()` return value to have the same value logged between "stopping" and "stopped" log entries.

Reply:
Good idea! The identifier wouldn't change with the current code, but some future-proofing in case we decide to null-out the transport in the future can't hurt.